client/components/wcc-settings-form/index.js

```diff
@@ -15,64 +15,84 @@ import ObjectPath from 'objectpath';
 import * as SettingsActions from 'state/settings/actions';
 import * as PackagesActions from 'state/form/packages/actions';
 
-const WCCSettingsForm = ( props ) => {
-	const {
-		layout,
-		saveFormData,
-		formActions,
-		noticeActions,
-	} = props;
-
-	const setIsSaving = ( value ) => formActions.setField( 'isSaving', value );
-	const setSuccess = ( value ) => {
-		formActions.setField( 'success', value );
+const WCCSettingsForm = React.createClass( {
```

Comment:
Is there something ES6 syntax cannot support? I'd rather not lose ES6 syntax if possible.